### PR TITLE
[FEAT] Replace SettingsController with AboutController

### PR DESCRIPTION
**Pull request by kingsqd**
In this PR 
I am creating a more controller for the more screen, and removing the old settings code.

Simulator Screen Shot - iPhone 11 Pro Max - 2020-06-06 at 12 13 01

### CODE REVIEW

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//
+//  MoreController.swift
+//  SayTheirNames
+//
+//  Created by Hakeem King on 6/5/20.
```
Comment (ohwhen) [inline SayTheirNames/Source/Controller/More/MoreController.swift:5]: Please include the new MIT license here.

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
+//
+//  MoreView.swift
+//  SayTheirNames
+//
+//  Created by Hakeem King on 6/5/20.
```
Comment (ohwhen) [inline SayTheirNames/Source/Controller/More/MoreView.swift:5]: Same (MIT License)

File: SayTheirNames/Source/Controller/More/MoreController.swift [hunk 3]
```diff
@@ -22,12 +22,21 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
+
```
Comment (evilpenguin) [inline SayTheirNames/Source/Controller/More/MoreController.swift:25]: Linter issue. Can we fix?

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+
```
Comment (evilpenguin) [inline SayTheirNames/Source/Controller/More/MoreView.swift:25]: Linter issue. Can we fix?

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+import UIKit
+
+/// The UI for More
+final class MoreView: UIView {
+
+    private lazy var moreLabel: UILabel = {
+        let label = UILabel()
+        label.text = "More"
```
Comment (evilpenguin) [inline SayTheirNames/Source/Controller/More/MoreView.swift:33]: Aren't we using localization or SwiftGen for this?

File: SayTheirNames/Source/Controller/More/MoreView.swift [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+// MARK: - Configurations
+private extension MoreView {
+
+    private func setupMoreLabel() {
+        addSubview(moreLabel)
+        moreLabel.fillSuperview()
+    }
+
+}
+
```
Comment (evilpenguin) [inline SayTheirNames/Source/Controller/More/MoreView.swift:70]: Linter issue. Can we fix?

**Comment by Sharkesm** [comment]
@kingsqd Can you resolve the conflicts

**Comment by kingsqd** [comment]
@Sharkesm done

**Comment by evilpenguin** [comment]
@kingsqd A few more conflicts now.

**Comment by kingsqd** [comment]
@ohwhen  I have addressed your changes. could you accept ?

**Pull request merged**